include/boost/container/container_fwd.hpp

```diff
@@ -115,7 +115,8 @@ template <class T, std::size_t Capacity>
 class static_vector;
 
 template < class T, std::size_t N
-         , class Allocator= new_allocator<T> >
+         , class Allocator= new_allocator<T>
+         , std::size_t alignment= 0 >
```

Comment:
Hi,
I've deliberately defaulted alignment to 0 to be able to also align the storage below its natural alignment. I don't have any applications for that right now, but since the dynamic allocator has the ability, I thought it could be an option...